content/efcore/getting-started/samples/defining-unique-identifiers/composite-key/README.md

```diff
@@ -0,0 +1,10 @@
+# Composite Key
+
+This code example demonstrates how to configure a composite key in EF Core via Fluent API.
+
+## Code Files:
+- [Program.cs](Program.cs)
+- [LibraryContext.cs](LibraryContext.cs)
+- [CheckoutRecord.cs](CheckoutRecord.cs)
+- [Book.cs](../../shared/Book.cs)
```

Comment:
This implementation of `Book` requires `Author`. Use an implementation that doesn't need `Author`